src/main/java/com/eternalcode/parcellockers/task/ParcelSendingTask.java

```diff
@@ -0,0 +1,11 @@
+package com.eternalcode.parcellockers.task;
+
+import org.bukkit.scheduler.BukkitRunnable;
+
+public class ParcelSendingTask extends BukkitRunnable {
+
+    @Override
+    public void run() {
+        
+    }
+}
```

Comment:
??

Reply:
Jajku karol jak widzisz taką pustą klasę to chyba wiadomo że tu dopiero coś będzie powstawało, totalnie zbędny komentarz.

Reply:
Kiedyś też tak zrobiłem - zrobiłem sobie schemat pustych klas/klasy, i dostałem komentarz na review, że takich pustych klas nie powinno się wrzucać na mastera, stąd mój komentarz.

Reply:
Masz rację, taski zrobię w nastepnym PR